rosidl_typesupport_opendds_c/rosidl_typesupport_opendds_c-extras.cmake.in

```diff
@@ -1,12 +1,17 @@
+# generated from
+# rosidl_typesupport_opendds_c/rosidl_typesupport_opendds_c-extras.cmake.in
```

Comment:
Since this code is committed, I don't think it's "generated". I would remove this comment to avoid the confusion or remove this file from the repo if it is generated.

Reply:
My guess was that this comment gets exported along with the content of this file.

Reply:
Then the generator should add this to the generated output file. The committed input file shouldn't say this.

Reply:
This input file will be expanded by cmake. I don't think it will generate these comments for us.

Reply:
Then it should definitely be removed from the committed code since the committed code is obviously not generated.

Reply:
I see this technique (hard coded comments intended for target generation) used also in template files that are source for expansion.